dags/nf-dynamic-challenge-dag.py

```diff
@@ -35,12 +35,13 @@ def launch_nf_dynamic_challenge_on_tower(**context):
         hook = NextflowTowerHook(context["params"]["tower_conn_id"])
         info = LaunchInfo(
             run_name=context["params"]["tower_run_name"],
-            pipeline="Sage-Bionetworks-Workflows/nf-dynamic-challenge",
+            pipeline="Sage-Bionetworks-Workflows/nf-synapse-challenge",
             revision="main",
             params={
                 "view_id": context["params"]["challenge_view_id"],
             },
             workspace_secrets=["SYNAPSE_AUTH_TOKEN"],
+            entry_name="DATA_TO_MODEL_CHALLENGE",
```

Comment:
Should this be dynamic challenge?

Reply:
This is the entrypoint for all data-to-model challenges. My hope is that we can use the same subworkflow for more than one individual challenge. This is just the name of that subworkflow.